hdw.dat.pgr

```diff
@@ -8,6 +8,10 @@
 #
 #  1   2      3        4         5       6      7      8    9   10   11   12  13(1)  13(2)  13(3)  14   15  16 17
 #
+# UNTIL 22/Sep/2019 18:00:00.0 Date when Borealis installed in pgr
+    6 2019 22917600  53.980 -122.590   670.0   -5.0  3.24  1   10  0.000  1    0.0 -100.0    0.0 0.0  0  225 16
```

Comment:
I think this `yrsec` corresponds to `20190923 06:00:00 UT`, rather than `20190922 18:00:00 UT` as the comment line indicates.  Should the `yrsec` instead be `22874400`?